llconf/entry.c

```diff
@@ -137,7 +137,7 @@ void _cnf_find_entry(struct cnfresult **pcr, struct cnfnode *cn_parent,
 		if(*p != ']')
 			return; // FIXME: error condition
 
-		index = atoi(tmp);
+		index = strtoi(tmp);
```

Comment:
This only works in `tdnf` because it has `strtoi()` in its libraries. If this code were copied to another project without `strtoi()` it will fail.

I don't think we need this here. `atoi()` is just fine.